bitnami/zookeeper/templates/statefulset.yaml

```diff
@@ -184,11 +184,14 @@ spec:
           configMap:
             name: {{ template "zookeeper.fullname" . }}
         {{- end }}
-        {{- if not .Values.persistence.enabled }}
+{{- if not .Values.persistence.enabled }}
```

Comment:
Please indent the templates at the same level as the code. This applies to the rest of the file